package.json

```diff
@@ -29,13 +28,14 @@
     "eslint-config-airbnb-typescript": "^17.1.0",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-import": "^2.28.1",
-    "hardhat": "^2.17.4",
+    "hardhat": "^2.22.6",
     "prettier": "^3.0.3",
     "prettier-plugin-solidity": "^1.1.3",
     "ts-node": "^10.9.1",
     "typescript": "^5.2.2"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "^4.9.3"
+    "@openzeppelin/contracts": "^4.9.3",
+    "dotenv": "^16.4.5"
```

Comment:
if `hardhat` is a dev dependency, `dotenv` should be a dev dependency as well

Reply:
Fix : Commit: ec39941fb2eb82da57e68c634bd2beffc0b79631